content/blog/r-mpi-part-one.md

```diff
@@ -0,0 +1,220 @@
+---
+author: "Wes Hinsley"
+date: 2023-06-22
+title: MPI from R - Part One
+best: false
+tags:
+ - R
+ - MPI
+ - HPC
+---
+
+[MPI](https://en.wikipedia.org/wiki/Message_Passing_Interface) is a 
+standard for allowing a team of processes to solve a problem together. Various
+implementations of that standard exist; [MPICH](https://www.mpich.org/) is
+perhaps the most well known. [Microsoft MPI](https://github.com/Microsoft/Microsoft-MPI)
+mimics MPICH on Windows, as does [Intel MPI](https://www.intel.com/content/www/us/en/developer/tools/oneapi/mpi-library.html#gs.1crzpx)
+on various platforms. Typically you'd then include these libraries in projects 
+written in C, C++, or Fortran.
+
+Furthermore, recent releases of [Rtools](https://cran.r-project.org/bin/windows/Rtools/)
+have included support for Microsoft MPI. Here, we're going to use that to write some
```

Comment:
I think it would be nice to say here that you'll also provide instructions for those playing along on Linux, since you do, helpfully!